Shared/ThresholdKeyView.swift

```diff
@@ -83,7 +87,7 @@ struct ThresholdKeyView: View {
                                 // Why does the application state Panic ??
                                 let key = try? await threshold_key.reconstruct()
                                 if key == nil {
-                                    alertContent = "Reconstruction failed"
+                                    alertContent = "Reconstruction failed. This is most likely caused by a non-zero value for required_share. "
```

Comment:
This message is incorrect and won't be reached since the try? in the above will fail.

Reply:
fixed message